src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -262,6 +264,13 @@ async function asObjectiveCClass(type, cluster, options) {
         hash: { preserveAcronyms: preserveAcronyms },
       });
     }
+    // Use a custom prefix if specified.
+    if (options.hash.structTypePrefix) {
+      return (
+        options.hash.structTypePrefix +
+        `${cluster}Cluster${appHelper.asUpperCamelCase(type)}`
+      );
+    }
```

Comment:
I believe this is the only "real" change in this diff and everything else is a whitespace change?

Reply:
That is correct. During git commit, Pretty quick reformat... introduced the whitespace changes